corehq/apps/export/forms.py

```diff
@@ -405,11 +406,12 @@ def get_edit_url(self, export):
         return reverse(EditNewCustomFormExportView.urlname,
                        args=(self.domain_object.name, export._id))
 
-    def get_form_filter(self):
+    def get_form_filter(self, mobile_user_and_group_slugs):
         return filter(None, [
             self._get_datespan_filter(),
-            self._get_group_filter(),
-            self._get_user_filter()
+            self._get_group_filter(mobile_user_and_group_slugs),
+            self._get_user_filter(mobile_user_and_group_slugs),
+            self._get_user_ids_filter(mobile_user_and_group_slugs)
```

Comment:
Makes sense.  I like a good prefactor with no functional changes before the feature work.